Mathlib/Analysis/SpecialFunctions/Trigonometric/Complex.lean

```diff
@@ -61,13 +61,15 @@ theorem sin_ne_zero_iff {θ : ℂ} : sin θ ≠ 0 ↔ ∀ k : ℤ, θ ≠ k * π
   rw [← not_exists, not_iff_not, sin_eq_zero_iff]
 #align complex.sin_ne_zero_iff Complex.sin_ne_zero_iff
 
+/-- Tangent of a complex number is equal to zero
+iff this number is equal to `k * π / 2` for an integer `k`.
+
+Note that this lemma takes into account that we use zero as the junk value for division by zero.
+See also `Complex.tan_eq_zero_iff'`.  -/
 theorem tan_eq_zero_iff {θ : ℂ} : tan θ = 0 ↔ ∃ k : ℤ, θ = k * π / 2 := by
```

Comment:
Could you take this occasion to swap the right hand side to `k * π / 2 = θ`? Same thing in the next lemma.

Reply:
Done (for tangent, not for other trig functions yet).